src/EnergyPlus/DataEnvironment.hh

```diff
@@ -22,6 +22,7 @@ namespace DataEnvironment {
 	extern Real64 const EarthRadius; // Radius of the Earth (m)
 	extern Real64 const AtmosphericTempGradient; // Standard atmospheric air temperature gradient (K/m)
 	extern Real64 const SunIsUpValue; // if Cos Zenith Angle of the sun is >= this value, the sun is "up"
+	extern Real64 const StdBaroPressSeaLevel; // Standard barometric pressure at sea level (Pa)
```

Comment:
if I cared, which I don't, I'd point out that the word choice of barometric pressure to me is something that varies with the weather and not the same as a standard atmosphere of press.  Like when a hurricane is coming the barometric pressure at sea level drops so it isn't a fixed value of Pa.  But since I don't care I 'll just merge to use whatever you want to call it.